website_partner_firstname/README.rst

```diff
@@ -0,0 +1,31 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```

Comment:
According to https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst you should add here another line of equal signs